ciphers/porta_cipher.py

```diff
@@ -0,0 +1,113 @@
+          ["X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W"]], "H": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W"]],
+    "I": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T", "U", "V"]], "J": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T", "U", "V"]],
+    "K": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T", "U"]], "L": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T", "U"]],
+    "M": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T"]], "N": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S", "T"]],
+    "O": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S"]], "P": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R", "S"]],
+    "Q": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R"]], "R": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q", "R"]],
+    "S": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q"]], "T": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P", "Q"]],
+    "U": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P"]], "V": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O", "P"]],
+    "W": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O"]], "X": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N", "O"]],
+    "Y": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"],
+          ["O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N"]], "Z": [["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"], ["O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z", "N"]]}
+
+
+def generateTable(key):
```

Comment:
`def generate_table(key):` is a more Pythonic function name.

Reply:
updated in latest